trainer/helper_functions.py

```diff
@@ -0,0 +1,64 @@
+import json
```

Comment:
If you are using headers, you should write them on all files.

Reply:
Sure, I will add header here as well.